web/assets/css/transmission-app.scss

```diff
@@ -499,12 +499,20 @@ a {
         font-size: 1em;
         font-weight: normal;
       }
+    }
 
-      &:not(.paused) {
+    &.selected {
+      &.torrent-name {
         color: var(--color-fg-primary);
       }
     }
 
+    &:not(.selected) .paused {
+      &.torrent-name {
+        color: var(--color-fg-disabled);
+      }
+    }
```

Comment:
This whole part can be simplified to:

```css
    &:not(.selected) {
      .torrent-name.paused {
        color: var(--color-fg-disabled);
      }
    }
```

There is no need to explicitly define `color: var(--color-fg-primary);` because it is already defined by the `html, body` selector.